content/slides/intro-to-jupyterlab.qmd

```diff
@@ -28,6 +33,43 @@ JupyterLab is not shared!
     * Your computations won't interfere with anyone else's
 :::
```

Comment:
I think some of these notes need to move now that we moved the server sizing content to the next slide.